testing/utils.go

```diff
@@ -1,7 +1,7 @@
 package ibctesting
 
 import (
-	"rand"
+	"math/rand"
```

Comment:
what does this do?

Reply:
imports the right thing :upside_down_face: